Provision Thanos EBSes

bboykov opened the pull request:
Prior to this change, there are no Thanos EBS provisioning tasks. The ones in use were created manually. This change adds tasks to provision Thanos EBSes during initial cluster creation.

Jira: https://jira.ft.com/browse/CPH-37
TODO:
- [x] This change needs to be tested before merging into master

bboykov committed:
Provision Thanos EBSes

Prior to this change, there are no Thanos EBS provisioning tasks. The
ones in use were created manually. This change adds tasks to provision
Thanos EBSes during initial cluster creation.

Jira: https://jira.ft.com/browse/CPH-37

mihaylovmihail reviewed (dismissed):
Seem fine.

emilangelov reviewed (commented):
Looks good, lets not merge it and hold the merge till we validate everything :)

bboykov committed:
Fix broken Ansible tasks

During the testing of the Thanos EBS provisioning some of the tasks were
failing. This commit fixes the issues.

mihaylovmihail reviewed (approved):
fine for me

emilangelov reviewed (approved):
kk, I don't see anything disturbing here.
